dev_docs/RELEASES.md

```diff
@@ -72,7 +72,9 @@ For patch releases, only the version on the existing major and minor version bra
 
 * A new [tag](https://github.com/elastic/apm-server/releases) will automatically be created on GitHub.
 
-* Bump the version in anticipation of the next release, e.g. [after 8.13.3 release](https://github.com/elastic/apm-server/pull/13066) bump to 8.13.4. **Prepare this PR ahead of time** but only merge it once pinged by the Release Manager on release date.
+* Merge the PRs created by the GitHub workflow ran as part of the steps in the ["Day after feature freeze"](#day-after-feature-freeze) section. 
```

Comment:
This only applies to patch release, not minor release, right?

Reply:
I agree and this needs to be clarified as there are a lot of PRs involved in this process and as we discovered recently other PRs (docs + version bumps) must not be merged before release date.

Reply:
What do we need to do to clarify? Reach out to the robots?

> This only applies to patch release, not minor release, right?

Good point, thanks for the callout. Should we make a distinction like we do above for minor/patch release?
E.g. say: "for minor releases, merge this and that; for patch releases, merge this other and that other PR".

Reply:
I think the old bullet point was correct, no? Basically on the release day the next patch version bump has to happen, everything else is supposed to already be done by the day after feature freeze or the day before release.

Reply:
It depends how we structure the flow, I think.
The old doc assumed the version bump PR is created on FF, and merged on release day.
I don't see why a time distance between the 2 events should exist, and I see more risk in creating and merging a PR for version bump _before_ we are requested to do so (as we would produce a DRA version that is too early).
That's why I added the "IMPORTANT" section below, but I also agree that the previous phrasing was more correct.
I'll amend that